client/tests/ReviewRating.test.jsx

```diff
@@ -112,9 +112,10 @@ describe('ReviewRating Component', () => {
 describe('sub-component in ReviewRating', () => {
   test('render Helpfulness Component', async () => {
     await act(async () => {
-      render(<Helpfulness
+      await render(<Helpfulness
```

Comment:
I am surprised you did not have to wrap this render in act(). Are you sure your component is rendering properly? I would use debug() to print out your render and check.